app/server/app.js

```diff
@@ -72,6 +73,16 @@ getWithConfig(app, "/collection/:collectionSlug", redirectCollectionHandler(), {
   logError
 });
 
+ampRoutes(app, {
+  seo: new SEO({
+    staticTags: STATIC_TAGS,
+    enableTwitterCards: true,
+    enableOgTags: true,
+    enableNews: true,
+    structuredData: STRUCTURED_DATA
+  })
+});
```

Comment:
can you move this to a separate function and reuse both places(ampRoutes and isomorphicRoutes)?

Reply:
Once structured data is done we can use the same  function for both